contracts/foundry-tests/RateChiMultioracle.t.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+
+import "@yield-protocol/utils-v2/contracts/access/AccessControl.sol";
+import "../oracles/compound/CompoundMultiOracle.sol";
+import "../mocks/oracles/compound/CTokenChiMock.sol";
+import "../mocks/oracles/compound/CTokenRateMock.sol";
+import "./utils/Test.sol";
+import "./utils/TestConstants.sol";
+
+contract RateChiMultiOracleTest is Test, TestConstants, AccessControl {
+    CTokenChiMock public cTokenChi;
+    CTokenRateMock public cTokenRate;
+    CompoundMultiOracle public compoundMultiOracle;
+
+    bytes6 public baseId = 0x25dde80ea598;
+    bytes6 public mockBytes6 = 0x000000000001;
+
+    function setUp() public {
+        cTokenChi = new CTokenChiMock();
+        cTokenRate = new CTokenRateMock();
+        compoundMultiOracle = new CompoundMultiOracle();
+        compoundMultiOracle.grantRole(0x92b45d9c, address(this));
+        compoundMultiOracle.setSource(baseId, CHI, address(cTokenChi));
+        compoundMultiOracle.setSource(baseId, RATE, address(cTokenRate));
+        cTokenChi.set(WAD * 2);
+        cTokenRate.set(WAD * 3);
+    }
+
+    function testRevertUnknownSource() public {
+        vm.expectRevert("Source not found");
+        compoundMultiOracle.get(mockBytes6, mockBytes6, WAD);
+    }
+
+    function testSetRetrieveChiRate() public {
```

Comment:
`chi` is just that, not `chiPrice`. Same for `rate`.

`chi` is the factor by which savings increase, and `rate` is the factor by which debts increase.

Reply:
Understood. Hopefully `getChi`/`peekChi` and `getRate`/`peekRate` will be sufficient for the differentiation.